colorschemes/abc-dark.conf

```diff
@@ -16,29 +16,29 @@
 # Free Software Foundation, Inc., 59 Temple Place - Suite 330,
 # Boston, MA 02111-1307, USA.
 #
-# Other:
-# GNU General Public License v2.0
-#https://github.com/geany/geany/blob/master/COPYING
-#https://github.com/geany/geany/blob/d9f8cdbad58d09f0c18ca8acccb49209263018f0/data/filedefs/filetypes.common#L2
-# Inspired by Github, Sublime Text, Geany
+# Other:  
```

Comment:
Why did you add trailing spaces on the following lines?

Reply:
I have now redone and tried to fix the space, hope so good